gpu/gpu_darwin.go

```diff
@@ -6,18 +6,30 @@ import "C"
 import (
 	"runtime"
 
-	"github.com/jmorganca/ollama/api"
+	"github.com/pbnjay/memory"
 )
 
 // CheckVRAM returns the free VRAM in bytes on Linux machines with NVIDIA GPUs
 func CheckVRAM() (int64, error) {
```

Comment:
You can remove the `error` returned by the function, it's not returned anymore

Reply:
The code is getting a bit tangled, but the linux/windows variants do return error for CPU scenarios, but it's become pedantic, so I think removing the error return is probably safe, just requires some changes to llm.go and routes.go

Reply:
Ah, dang, maybe a different change in that case

Reply:
I'll leave it for this PR but worth revisiting